README.md

```diff
@@ -490,7 +490,6 @@ These are all the code changes! 🎉
 ## 1. Installation
 
 ```bash
-export ACRA_DOCKER_IMAGE_TAG='master'
```

Comment:
uh oh, i missed this line when updated Readme after release :(

thanks for catching!